en/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: news_post
+title: "Multiple vulnerabilities in RubyGems"
+author: "usa"
+translator:
+date: 2017-08-29 12:00:00 +0000
+tags: security
+lang: en
+---
+
+There are multiple vulnerabilites in RubyGems bundled by Ruby.
+It is [reported at the official blog of RubyGems](http://blog.rubygems.org/2017/08/27/2.6.13-released.html).
+
+## Details
+
+The following vulnerabilites have been reported.
+
+* a DNS request hijacking vulnerability
+* an ANSI escape sequence vulnerability
+* a DOS vulernerability in the query command
+* a vulnerability in the gem installer that allowed a malicious gem to overwrite arbitrary files
+
+It is strongly recommended for Ruby users to take one of the following workarounds as soon as possible.
+
+## Affected Versions
+
+* Ruby 2.2.7 and all versions of Ruby 2.2 prior to it
```

Comment:
"2.2.0 - 2.2.7" is enough? "and all versions of Ruby 2.x" looks too verbose to me.